src/components/Pagination/Pagination.js

```diff
@@ -1,17 +1,15 @@
 import React from "react";
 import "./Pagination.css";
-import ChevronLeftIcon from "@material-ui/icons/ChevronLeft";
-import ChevronRightIcon from "@material-ui/icons/ChevronRight";
 
```

Comment:
these icons are not needed? @Justinnn07

Reply:
I am making the UI of the pagination little attractive .. moreover, the prev button was not working tooo!